docs/development.md

```diff
@@ -130,6 +130,30 @@ git add glide.yaml glide.lock vendor
 git commit
 ```
 
+#### Updating OpenShift
+
+1. Update the OpenShift version in `scripts/vendor-openshift.sh`.
```

Comment:
Include link to the file (github link?)

Perhaps 
```sh
Update OPENSHIFT_VERSION within [`scripts/vendor-openshift.sh`](https://github.com/kedgeproject/kedge/blob/master/scripts/vendor-openshift.sh)
```

Reply:
done!